PKHeX.Core/Resources/text/script/const_pt_es.txt

```diff
@@ -1,5 +1,5 @@
 0048	Inicial	0:Sin Elegir,387:Turtwig,390:Chimchar,393:Piplup
-0122	Togepi Egg	0:Visited Statue,1:Received HM01,2:Cynthia Disappeared,3:Cynthia Gives Egg,4:Egg Pending,5:Egg Received
+0122	Togepi Egg	0:Visited Statue,1:Received HM01,2:Cynthia Desaparecio,3:Cynthia da un huevo,4:Huevo Pendiente,5:Huevo Recibido
```

Comment:
Thanks for your contribution!  In this line, should "Visited Statue" and "Received HM01" also be translated?

Reply:
You are welcome ! Yes as I said above in the commentary there are still some untranslated texts in English, if my collaboration seems ok and is accepted I will continue with the translation of the missing texts.

Reply:
Ah, I was wondering why only part of that line was translated.  Go ahead and do the others you were planning.  You may want to keep an eye on the capitalization to stay consistent ("Huevo Recibido" vs "Huevo recibido" and "No activado" vs "No Activado").